README.adoc

```diff
@@ -178,20 +178,20 @@ InventoryReadinessCheck.java
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[]
 ----
 
-This time, you are checking whether or not the service is in maintenance or if it's down.
-For simplicity, the custom [hotspot=3 file=2]`io_openliberty_guides_inventory_inMaintenance`
+In the [hotspot=isHealthy file=0]`isHealthy()` method, 
+you will report the `inventory` service as not ready if the service is in maintenance or if its dependant service is unavailable.
+
+For simplicity and simulation, the custom [hotspot=3 file=2]`io_openliberty_guides_inventory_inMaintenance`
```

Comment:
Suggested update

>For simplicity, the custom `io_openliberty_guides_inventory_inMaintenance`
MicroProfile Config property, which is defined in the `resources/CustomConfigSource.json`
file, is used to indicate whether the service is in maintenance.

Reply:
updated